src/status_im/chat/models.cljs

```diff
@@ -295,16 +245,17 @@
               (when-not (group-chat? cofx chat-id)
                 (transport.filters/load-chat chat-id))
               (when platform/desktop?
-                (mark-messages-seen chat-id))
+                (message-seen/mark-messages-seen chat-id))
               (when (and (one-to-one-chat? cofx chat-id) (not (contact.db/contact-exists? db chat-id)))
-                (contact.core/create-contact chat-id)))))
+                (contact.core/create-contact chat-id))
+              (loading/load-messages))))
 
 (fx/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
   [cofx chat-id]
   (fx/merge cofx
-            (navigation/navigate-to-cofx :chat {})
-            (preload-chat-data chat-id)))
+            (preload-chat-data chat-id)
```

Comment:
Is the order important here? Maybe we should leave a comment if so.

Reply:
re-frame doesn't guaranty order of fx handlers, so I did it just in case :)